service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/util/SectionFilterer.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.adaptors.oneoneone.cda.report.util;
+
+import org.apache.commons.lang3.ArrayUtils;
+import uk.nhs.connect.iucds.cda.ucr.CE;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Component5;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Section;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SectionFilterer {
```

Comment:
Class not used at all.

Reply:
Thomas and I was doing the same Section filtering and I decided to extract these out late last night. I will remove for now.

Reply:
Module removed...